src/core/optimization_container.jl

```diff
@@ -671,7 +671,7 @@ function build_impl!(
 
     TimerOutputs.@timeit BUILD_PROBLEMS_TIMER "Construct $(transmission)" begin
         @debug "Building $(transmission) network formulation"
-        construct_network!(optimization_container, sys, transmission)
+        construct_network!(optimization_container, sys, transmission, template)
```

Comment:
we need to get rid of the instances where we pass the template. This is no longer needed.

Reply:
this is still needed for the `PTDFPowerModel` formulation which builds the "sparse" PTDF with PowerModels. It needs the template to understand which branches it should create PTDF rows for. I'm all for alternatives, but this is what I've been able to come up with.

Reply:
ok, we can refactor this when we review the container construction